src/lib/y2journal/query.rb

```diff
@@ -94,11 +94,14 @@ def self.boots
         # (slightly stripped down, see test/data for full-length examples)
         # -1 a07ac0f240 Sun 2014-12-14 16:50:09 CET—Mon 2015-01-26 19:18:43 CET
         #  0 24a9a83ecf Mon 2015-01-26 19:55:33 CET—Mon 2015-01-26 20:05:16 CET
-        if line.strip =~ /^\s*(-*\d+)\s+(\w+)\s+(.+)$/
+        if line.strip =~ /^\s*(-*\d+)\s+(\w+)\s+(.+)—(.+)$/
```

Comment:
Please check the documentation of this method. It says "with all the keys being symbols and all the values being strings". But that's not true anymore after your changes.

Reply:
doc comment is updated